pkgs/development/web/bun/update-script

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq gawk
+# vim:ft=bash
```

Comment:
Is this needed?

Reply:
Mode is for emacs. IDK if vim respects it too

Reply:
I have seen such things in Python code from a long time ago.

I did a quick research, and there is a vim plugin for it.